nginx.conf.template

```diff
@@ -2,6 +2,9 @@ server {
     listen 3046;
     server_name localhost;
 
+    # Content Security Policy (CSP) headers
+    add_header 'Content-Security-Policy' "default-src 'self'; script-src 'self'; style-src 'self'; img-src 'self' data: ws://localhost:3345 data: image/gif video/; font-src 'self' data:; frame-src 'self'; object-src 'none'; base-uri 'self';";
```

Comment:
Two things to do for now:
1. Please replace the `ws://localhost:3345` with some var.
2. Please verify if the `unsafe-inline` is really needed - it's better to avoid it whenever possible.

Reply:
I've made changes to set CSP in the same way that currently works for nuxt-client.
I could not get rid of `unsafe-inline` for styles. We do not set inline styles on our own but external packages do. That's probably why it is also set on nuxt-client.